### DISCUSSION TITLE
Weird behavior of top 3 holes in transverse flute

### DISCUSSION THREAD

**Discussion by edwardkort** [edwardkort/WWIDesigner#96] [Flute Study] [answered]
From Bamboo Flute:
I have tried that flute model. I did 2 months practice on it but it behave weird for top three holes. Have anybody built transverse flute using this application? I respect all American flutes but Indian bansuri type transverse flute can play meend which is not possible using any western or American flutes. Indian Classical music is not possible without meend. American flutes and western fluted are jumping from one note to another which is easy to make but flutes having ability to play meend required an extraordinary skill because it also defines the tone in between the notes.

**Comment by bhp1** [comment, marked as answer]
WIDesigner recognizes a hole as either open or closed; it isn't equipped to model in-between states like half-holing, common in Western traditions, or meend.  I don't know what characteristics of a hole would make meend harder or easier to execute.
